BUILDING_ON_WINDOWS.md

```diff
@@ -28,15 +29,26 @@ Notes:
 #### When prompted which components to install:
 
 1. Unfold the tree element that says "Qt"
-2. Unfold the top most tree element (latest stable Qt version, e.g. `Qt 5.15.2`)
+2. Unfold the top most tree element (latest stable Qt version, e.g. `Qt 6.5.3`)
```

Comment:
If we're advising this version, we should update the third and fourth point as well:

```
3. Under this version, select the following entries:
   - `MSVC 2019 64-bit` (or alternative version if you are using that)
   - `Qt 5 Compatibility Module`
   - `Additional Libraries` > `Qt Image Formats`
4. Under the "Developer and Designer Tools" tree element (at the bottom), ensure that `Qt Creator X.X.X` and `Debugging Tools for Windows` are selected. (they should be checked by default)
```

Reply:
Fixed in [`5f1a46c` (#4858)](https://github.com/Chatterino/chatterino2/pull/4858/commits/5f1a46c4e8ecf8f7421187f75d76dbc33c1974e8) and [`87bc2ac` (#4858)](https://github.com/Chatterino/chatterino2/pull/4858/commits/87bc2ac4a303122e9e5f4dad9d4ce499b7191cf8)